Sources/KituraWebSocket/WebSocketConnection.swift

```diff
@@ -144,7 +148,12 @@ extension WebSocketConnection: ChannelInboundHandler {
                     case .binary:
                         fireReceivedData(data: frame.data.getData(at: 0, length: frame.data.readableBytes) ?? Data())
                     case .text:
-                        fireReceivedString(message: frame.data.getString(at: 0, length: frame.data.readableBytes) ?? "") 
+                        if let text = frame.unmaskedData.getString(at: 0, length: frame.unmaskedData.readableBytes, encoding: .utf8) {
+                            fireReceivedString(message: text)
+                        } else {
+                            connectionClosed(reason: .dataInconsistentWithMessage, description: "Failed to convert received payload to UTF-8 String")
```

Comment:
The Kitura-WebSocket tests expect this error code to be 1005 i.e `WebSocketErrorCode(codeNumber: 1005)`